index.bs

```diff
@@ -91,8 +91,9 @@ we've prepared [a list of these questions in Markdown](https://raw.githubusercon
 </h3>
 
 Just because information can be exposed to the web doesn’t mean that it
-should be. How does exposing this information to an origin benefit a user?
-Is the benefit outweighed by the potential risks?  If so, how?
+should be. How does exposing this information to an origin or other party 
```

Comment:
Could you rebase? I don't think this change is necessary given the current text.